src/envs/behavior.py

```diff
@@ -369,15 +380,14 @@ def _ig_object_to_object(self, ig_obj: "ArticulatedObject") -> Object:
         ig_obj_name = self._ig_object_name(ig_obj)
         return Object(ig_obj_name, obj_type)
 
-    @functools.lru_cache(maxsize=None)
     def object_to_ig_object(self, obj: Object) -> "ArticulatedObject":
         """Maintains a mapping of objects to underlying igibson objects."""
         return self._name_to_ig_object(obj.name)
 
-    @functools.lru_cache(maxsize=None)
```

Comment:
can we add comments here mentioning that it's really bad for anyone to _ever_ add caching to these functions?